### PR TITLE
New helm notes command to display notes provided by a release's chart

### LINKED ISSUE #2742: refetch notes
Notes for the chart often contains useful information that you may need to revisit. Unless the user knows to save the note output, it can be lost currently.

There should be a command such as:
helm notes <release name>

That will redisplay the notes for a given release.

### PR DESCRIPTION
**Pull request by adshmh**
feat(helm): added new helm notes command to display notes provided by the chart of a release.
Fixes #2742 

Signed-off-by: Arash Deshmeh <adeshmeh@ca.ibm.com>

### CODE REVIEW

**Comment by thomastaylor312** [comment]
Hey @adshmh we'll be reviewing this soon! Could you please rebase a rebuild your docs against the new changes?

**Comment by adshmh** [comment]
@thomastaylor312 thank you. Rebased.

**Review by thomastaylor312 — CHANGES_REQUESTED** [review]
Hey @adshmh this is awesome! Can you move the notes command to be a subcommand of `helm get`? All of the rest of the commands for fetching values are there, so it seems like this belongs there as well. Otherwise, this looks good to go

**Review by bacongobbler — CHANGES_REQUESTED** [review]
fully agree with @thomastaylor312 WRT moving this under `helm get` for consistency with other commands like `helm get values`. Looks good otherwise!

**Comment by adshmh** [comment]
Thank you for the reviews. I have updated the PR to make `notes` a subcommand of `helm get`.

File: docs/helm/helm_notes.md [hunk 1]
```diff
@@ -0,0 +1,44 @@
+## helm notes
```
Comment (bacongobbler) [inline docs/helm/helm_notes.md]: Would you mind deleting this documentation? Doesn't look like it was cleaned up by `make docs`.

**Review by bacongobbler — CHANGES_REQUESTED** [review]

File: docs/helm/helm_notes.md (hunk 1 above)
Comment (adshmh) [reply docs/helm/helm_notes.md]: Thank you for the review. I have updated the PR.

**Review by bacongobbler — APPROVED** [review]
thanks so much for your patience. LGTM!

**Comment by bacongobbler** [comment]
Merging since @thomastaylor312's comments were addressed :)

**Comment by hickeyma** [comment]
v3 port created: #6619

**Pull request merged**